diagnostics/networking.sh

```diff
@@ -1,40 +1,19 @@
 #! /bin/bash
-if (($(id -u) != 0))
-then
-    printf "If you see 'ping: socket: Operation not permitted' errors, "
-    printf "run this command as root.\n"
-fi
-
 set -xu
 
 # Gather distro & kernel info.
 lsb_release -a || cat /etc/issue /etc/os-release
 uname -a
 
-# Output adapter & routing configuration.
+echo "Printing adapter & routing configuration"
 ip a
 ip route show table all
 ip neighbor
 ip link
 
-# Validate that the gateway is responsive and can route ICMP correctly.
-if gateway=$(ip route show | awk '/default/ { print $3 }'); then
-    ping -c 4 "$gateway"
-else
-    echo 'No gateway found.'
-fi
-
-ping -c 4 1.1.1.1
+# This will include the HTTP proxy env variables, if present
+echo "Printing all environment variables"
+printenv
```

Comment:
We should avoid printing the environment here because it might contain secrets. If you want the proxy variables I recommend printing whether the env variables are set or not but not their content

Reply:
thanks for pointing it out. will update to print if HTTP_PROXY, http_proxy and the other http proxy variables are set or not

Reply:
changed to print if the proxy variables are set or not